shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -698,6 +700,15 @@ public void onBackInvoked() {
           }
           : null;
 
+  @Override
+  public void setFrameworkHandlesBacks(boolean frameworkHandlesBacks) {
```

Comment:
suggestion: setFrameworkHandlesBack (without the plural s)?

Reply:
Maybe something to indicate it registers/unregisters a callback based on `frameworkHandlesBacks` since it isn't actually setting that, e.g. `toggleOnBackRegistry`?

Reply:
I'm going to go with `setFrameworkHandlesBack` for now.

I kind of wanted to be generic about what is actually happening under the hood in case that changes, or in case we ever needed to do this on another platform that handles it differently.  I think `setFrameworkHandlesBack` works well enough since it at least sounds declarative, like it's setting some state that decides what should happen to back gestures.

Let me know if anyone has other thoughts though.